src/main/java/com/michelin/cio/hudson/plugins/maskpasswords/MaskPasswordsOutputStream.java

```diff
@@ -119,11 +120,11 @@ public MaskPasswordsOutputStream(OutputStream logger, @CheckForNull Collection<S
 
     @Override
     protected void eol(byte[] bytes, int len) throws IOException {
-        String line = new String(bytes, 0, len);
+        String line = new String(bytes, 0, len, "UTF-8");
```

Comment:
This is probably incorrect; in general you would need to pay attention to the `Run.charset`. However the code was already wrong, as it used the default encoding on master.